python/lsst/cp/pipe/ptc.py

```diff
@@ -446,12 +413,42 @@ def runDataRef(self, dataRef, visitPairs):
 
             butler.put(linearizer, datasetType='Linearizer', dataId={'detector': detNum,
                        'detectorName': detName, 'calibDate': calibDate})
-
-        self.log.info(f"Writing PTC data to {dataRef.getUri(write=True)}")
+        rerunDir = list(dataRef.getButler().storage.repositoryCfgs.keys())[0]
```

Comment:
This seems like a lot of work for something that the butler should be handling.  I also don't think that this will work in gen3.

Reply:
Do we need this line then?  {{dataRef.getUri(write=True)}} fails. This line is printing the location of the dataset pickle file. Is there another way of getting this, if we decide that we need to print this information?

Reply:
I don't think we need to say where we're writing things.  A simple "Writing PTC data." will note that it's done with the calculation.